client/src/components/SiteList.js

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect, useState } from "react";
+import SiteTile from "./SiteTile";
+
+const SiteList = props => {
+    const [siteList, setSiteList] = useState([])
+
+    const getSites = async () => {
+        try {
+            const response = await fetch("/api/v1/sites")
+            if (!response.ok) {
+                const errorMessage = `${response.status} (${response.statusText})`
+                const error = new Error(errorMessage)
+                throw(error) 
+            }
+            const responseBody = await response.json()
+            setSiteList(responseBody.siteList)
+        } catch (error) {
+            console.error(`Error in Fetch: ${error.message}`)
+        }
+    }
+
+    useEffect(() => {
+        getSites()
+    }, [])
+
+    const sitesToRender = siteList.map(site => {
+        return(
+            <SiteTile
+                key={site.id}
+                name={site.name}
+                address={site.address}
+                description={site.description}
+                environment={site.environment}
+                minimumAge={site.minimumAge}
+            />
+        )
+    })
+
+    return(
+        <div>
+            <h1>Welcome to Boston!</h1>
+            <ul>
```

Comment:
each SiteTile is returning a collection of HTML elements wrapped in a fragment - when using an unordered list, `ul`, it will expect list item, `li`, elements as its direct children descendents. Here we should either remove the `ul` entirely because it is not being used for styling (could use a div if there was styling added), or make sure that the SiteTile returns a `li` as the top level element